test/libsolidity/syntaxTests/inheritance/allow_empty_duplicated_super_constructor_call.sol

```diff
@@ -0,0 +1,3 @@
+contract A { constructor() public { } }
+contract B1 is A { constructor() A() public {  } }
+contract B2 is A { constructor() A public {  } }
```

Comment:
Note to myself: modifier-style + no parentheses should be an error.

Reply:
Moved this to a separate issue #3838, since this pull request is already quite massive.